package.json

```diff
@@ -41,26 +42,26 @@
   },
   "homepage": "https://github.com/fastify/fastify-swagger#readme",
   "devDependencies": {
-    "@types/node": "^17.0.0",
-    "fastify": "^3.7.0",
-    "fastify-basic-auth": "^2.1.0",
-    "fastify-helmet": "^5.0.3",
+    "@types/node": "^17.0.13",
+    "fastify": "^3.27.0",
+    "fastify-basic-auth": "^2.2.0",
+    "fastify-helmet": "^7.0.1",
```

Comment:
Need replacing with `@fastify/basic-auth` and `@fastify/helmet`, and then any `requires()` of them.

Reply:
Done.